apollo-router/Cargo.toml

```diff
@@ -188,7 +188,7 @@ regex = "1.10.3"
 reqwest.workspace = true
 
 # note: this dependency should _always_ be pinned, prefix the version with an `=`
-router-bridge = "=0.5.21+v2.7.5"
+router-bridge = "=0.5.23+v2.8.0-alpha.0"
```

Comment:
Are we happy releasing a router which is using a federation alpha release? Is the intent to update this before we release the next version of the router?

Reply:
Yes. This is just to let us bake in a bit more testing before we officially release 2.8.0, but if all goes well we can just promote it. We'll certainly release and bump the pin before we cut a 1.48.0 rc